test/scenario/metering-plans/standard-services-hours/.gitignore

```diff
@@ -0,0 +1,3 @@
+node_modules/**
```

Comment:
I find the following .gitignore content better.

```
/node_modules
/.nyc_output
/coverage
/yarn.lock
```

Also notice that I have added the `yarn.lock` entry. I think we decided not to submit those (in sub-modules), and have them covered by the main `yarn.lock`.